nuxeo-features/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/jaxrs/batch/BatchManagerFixture.java

```diff
@@ -180,12 +178,9 @@ public void testAddChunkStream() throws IOException {
         String chunk2 = "é composé de ";
         String chunk3 = "3 chunks";
         long fileSize = fileContent.getBytes().length;
-        bm.addStream(batchId, "0", new ByteArrayInputStream(chunk1.getBytes("UTF-8")), 3, 0, "Mon doc.txt",
-                "text/plain", fileSize);
-        bm.addStream(batchId, "0", new ByteArrayInputStream(chunk3.getBytes("UTF-8")), 3, 2, "Mon doc.txt",
-                "text/plain", fileSize);
-        bm.addStream(batchId, "0", new ByteArrayInputStream(chunk2.getBytes("UTF-8")), 3, 1, "Mon doc.txt",
-                "text/plain", fileSize);
+        bm.addBlob(batchId, "0", Blobs.createBlob(chunk1), 3, 0, "Mon doc.txt", "text/plain", fileSize);
+        bm.addBlob(batchId, "0", Blobs.createBlob(chunk3), 3, 2, "Mon doc.txt", "text/plain", fileSize);
+        bm.addBlob(batchId, "0", Blobs.createBlob(chunk2), 3, 1, "Mon doc.txt", "text/plain", fileSize);
```

Comment:
Do we have a constant for "text/plain"?

Reply:
Probably. I was just refactoring another part of the test here though.